.ci/validation.sh

```diff
@@ -1186,6 +1186,63 @@ check-wildcards-on-pitest-target-classes)
   exit "$CLASSES_NO_WILDCARD_COUNT"
   ;;
 
+verify)
+  mvn -e --no-transfer-progress clean verify
+  ;;
+
+package-all-jar)
+  mvn -e --no-transfer-progress clean package -Passembly
+  ;;
+
+website-only)
+  mvn -e --no-transfer-progress clean site -Pno-validations
+  ;;
+
+pmd)
+  mvn -e --no-transfer-progress clean test-compile pmd:check
+  ;;
+
+spotbugs)
+  mvn -e --no-transfer-progress clean test-compile spotbugs:check
+  ;;
+
+checkstyle)
+  mvn -e --no-transfer-progress clean compile antrun:run@ant-phase-verify
+  ;;
+
+forbiddenapis)
+  mvn -e --no-transfer-progress \
+    clean compile test-compile forbiddenapis:testCheck@forbiddenapis-test
+  ;;
+
+test-class)
+  if [[ -z "$2" ]] ; then
+    echo "Error: test class is not defined."
+    echo "Example: mvn -e --no-transfer-progress clean test -Dtest=XdocsPagesTest,XdocsJavaDocsTest"
+    exit 1
+  fi
+  mvn -e --no-transfer-progress clean test -Dtest="$2"
+  ;;
+
+test-method)
```

Comment:
This can also be used for test classes, right?

Reply:
Yes, but from users perspective it is different

Reply:
please suggest better name, if no ideas for now, lets merge and after usage for some time, ti will come.
different names will help easily to grep.

Reply:
`run-test`?

Reply:
What will be name for second command?
They should be similar in naming.
run-test
run-test-method
?

Reply:
The parameter is used the same way, why do we need two commands?

Reply:
combined to "run-test"